War Engines first pass (#28)
GameBrains/er-core: modules/special-rules/pages/war-engines-in-close-combat.adoc

== War Engine assault moves and charges

You may move your War Engine units in the Assault phase, and you may make charge moves with them if you wish.
This does not affect the War Engine unit's ability to shoot in the Shooting phase in any way; it always fires at full effect.

War Engine units have a comparatively high Assault value, as you will see on their data sheets.
Lesser units can and must 'gang up' on a War Engine if they are to defeat it in close combat.

> You can place as many units in base contact with the War Engine unit as its initial Damage Capacity value.

Reviewer: ```suggestion
The maximum number of units that you may place in base contact with a War Engine unit is equal to the number of its initial Damage Capacity value.
```
...though see my next comment about the actual rule change.

> [NOTE.e40k]
> ====
> In _Epic 40,000_, you could only 'gang up' with half as many units, which meant that smaller War Engine units like Baneblades could only be attacked by two units -- the same as a regular target!
> ====

Reviewer: Given that strictly speaking this weakens war engines to some extent, is there any other reason for this rule change? 

I ask because I imagine that others may not see "we can't 'gang up' any more on small war engines than we can on regular units" as a problem — or not enough of a problem to justify doubling the number of units that can be in contact for all war engines.

Meantime, just a few minor tweaks:

```suggestion
[NOTE.e40k]
====
In _Epic 40,000_, you could only 'gang up' with half as many units as the initial Damage Capacity.
This meant that against small War Engine units such as the Baneblade you could only place two units in base contact -- the same as against a Vehicle or Infantry target!
====
```

Author: The previous discussion appeared to be strongly in favour of increasing it: https://www.facebook.com/groups/1004227353047038/posts/2267032236766537/

But I've started the usual polls to see if people disagree...

Reviewer: Some further tweaks: You make a good case on Facebook, and I agree — but I think that the particular point you used in this note was one of the weakest ones. While we don't want to go into depth here, I think that it'd be best to allude to the stronger reasons.

Something like this perhaps:

```suggestion
[NOTE.e40k]
.'Ganging up' on War Engine units
====
In _Epic 40,000_, you can only 'gang up' with *half* as many units as the initial Damage Capacity.
This leads to a number of odd effects, not least that it makes war engines *excessively* difficult to overcome in close combat -- so, we reference the full initial Damage Capacity instead.
====
```

> //{blank}[TODO: How do War Engine units -- or a mix of War Engine and regular units -- gang up on on other War Engine units?]
> // IJW reply just treat them as units, so that you can actually gang up on similar-size targets.

Reviewer: Yup, that makes sense -- I had some thought in mind while I was editing here, about how to apply the DCs on each side, but that was erroneous. Queuing for removal.
```suggestion
```

== Close combat attacks with a War Engine

War engines can also do a lot more damage in close combat than regular units.

* Roll a number of dice to hit equal to half the current Damage Capacity of the War Engine unit (round up).

> * As normal, you can apply hits to any enemy unit in the combat which is in base contact with any of your units.

Reviewer: ```suggestion
* As normal, you can allocate hits to any enemy units that are in base contact with any of your units in the close combat.
```

> [NOTE.e40k]
> ====
> In _Epic 40,000_, the rules for close combat attacks with War Engines assumed that you had a single War Engine in combat with the enemy, and that there were no other War Engines or allied detachments in the close combat. This revised version treats all the close combat hits normally, instead of limiting the hits to units in base contact with that specific War Engine.
> ====

Reviewer: Some tricky phrasing again — please check that I don't 'break' anything with this suggestion.
```suggestion
[NOTE.e40k]
====
In _Epic 40,000_, the rules for close combat attacks with War Engines are worded on the basis of a single War Engine in close combat with the enemy, with no other help.
In the absence of further clarification, this tends to imply that the allocation of hits might be on a per War Engine basis, in contrast to the usual rules.
We word the rules to make it clear that we should allocate all close combat hits in the same way, regardless of origin.
====
```

== Close combat attacks against a War Engine

* For each regular unit that you have in base contact with an enemy War Engine, roll a single hit die as normal.
* Ignore power fields and void shields for any hits that you score in close combat -- they damage the War Engine directly.
Roll for critical damage as normal.

> == War Engine detachments in close combat
> 
> * When a War Engine detachment is in close combat and some of the War Engine units are not in base contact with the enemy, do not add +1 for the units not in base contact.
> Instead, add a number equal to half the current Damage Capacity of these War Engines.
> This does not prevent the War Engine units from supporting other close combats.
> 
> [NOTE.e40k]
> ====
> In _Epic 40,000_, there were rules for War Engines to support *other* close combats, but no rules for supporting a close combat that the War Engine detachment was in. We have adapted the rules for supporting other close combats.
> ====
> 

Reviewer: Some tweaks — please check that I don't 'break' anything.
```suggestion
== War Engine detachments in close combat

* When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.
+
To be clear, such units may still also support *other detachments* as we explain at xref:war-engines-in-support-of-close-combats.adoc[].

[NOTE.e40k]
====
In _Epic 40,000_, the rules specify how War Engine units can support other detachments that are in close combat, but not how they can provide supporting fire within their own detachment.
We make both situations plain.
====

```

Reviewer: This appears to contradict other statements and is thus 'wrong': `This does not prevent the War Engine units from supporting other close combats.`

The original rules state `In order to lend its support to a close combat a war engine must be within 15cm of an enemy unit which is fighting in, or lending support to, the combat and the war engine can't be in close combat itself.` ...and our current Remastered rules mirror this with `To do this, your War Engine unit must not be engaged in combat itself`.

So, I'll tweak my suggestion here a bit further, thus: (along with some other further small tweaks while I'm at it)

```suggestion
== War Engine detachments in close combat

When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.

[NOTE.e40k]
.Supporting fire from War Engine units 
====
_Epic 40,000_ gives rules for War Engine units in support of other detachments that are in close combat -- but does not gives rules for how they can provide supporting fire within their own detachment.
We make both situations plain.
====

```

Author: > This appears to contradict other statements and is thus 'wrong': `This does not prevent the War Engine units from supporting other close combats.`

Could you explain in a bit more detail? I can't see where this is contradicting other statements. As a reader, I no longer know how this affects War Engines supporting other close combats.

Reviewer: The detail I already added above covers it, doesn't it?

> The original rules state `In order to lend its support to a close combat a war engine must be within 15cm of an enemy unit which is fighting in, or lending support to, the combat and the war engine can't be in close combat itself.` ...and our current Remastered rules mirror this with `To do this, your War Engine unit must not be engaged in combat itself`.

...where original is [War engines supporting close combats](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/WarEnginesSupportingCloseCombats.htm) and Remastered is [War engines in support of close combats](https://epicremastered.com/core/special-rules/war-engines-in-support-of-close-combats.html).

Note "can't be in close combat itself" and "must not be engaged in combat itself", respectively. Unless I'm missing something daft (always possible!) these seem to directly contradict that particular line that I've excluded from my suggestion.

Author: I think the missing bit is that close combats are resolved sequentially rather than simultaneously.

So a War Engine detachment can be in a close combat (with one of them outside base contact), resolve their own close combat, and then be in a position to support another separate close combat that hasn't been resolved yet. 

Without the removed bullet, it's unclear whether the 'supporting' War Engine can support its own detachment and then support the separate close combat.

Reviewer: Ah, okay. So, resolve all close combats, but choose the order carefully to maximise support from war engines... don't treat close combats as simultaneous.

I made an implicit assumption that although of course we resolve them one at a time, we nonetheless treat them as simultaneous. As we are **not** supposed to do that, **should we make this more clear somewhere**? (Others might make the same assumption that I did.) Or is it already clear somewhere and I've just overlooked/forgotten statements to that effect?

Latest iteration of the suggestion:

```suggestion
== War Engine detachments in close combat

When you add up Assault values for a War Engine detachment that is in close combat, do not add +1 for 'supporting fire' from each unit in the detachment that is within 15 cm of the enemy but not in base contact.
Instead, add a number equal to half the current Damage Capacity of each such War Engine unit.

To be clear, such units may still also support *other detachments* as we explain at xref:war-engines-in-support-of-close-combats.adoc[].

[NOTE.e40k]
.Supporting fire from War Engine units 
====
_Epic 40,000_ gives rules for War Engine units in support of other detachments that are in close combat -- but does not give similar rules to apply the same logic for supporting fire within their own detachment (so only the usual +1 would apply, which seems odd).
We make both situations plain.
====
```

Reviewer: @IJW-Wartrader Any thoughts on my last comment on this thread? If not then I'll just apply this suggestion and proceed to final checks with a view to publish, as there are no other outstanding items.

Author: @digi-brain that looks OK.

> I made an implicit assumption that although of course we resolve them one at a time, we nonetheless treat them as simultaneous. As we are **not** supposed to do that, **should we make this more clear somewhere**? (Others might make the same assumption that I did.) Or is it already clear somewhere and I've just overlooked/forgotten statements to that effect?

Hmm. Not sure. For me, 'The player with the Initiative decides the order in which to resolve close combats.' from close-combat-procedure.adoc makes it clear that you resolve one CC, then move onto the next and resolve that, etc.

Author: > Well, my point is that **of course** you must 'decide the order' to resolve one at a time, because that is the **only** practical way to do it in a tabletop wargame. So, by itself, this:
> 
> > The player with the Initiative decides the order in which to resolve close combats.
> > ...
> > To resolve a close combat, use the close combat procedure as follows.
> 
> ...doesn't preclude the possibility that we treat the set of combats that a player triggers as simultaneous events.

If you think it's unclear, then sure, add something. But I may not have explained myself properly.

<img width="745" alt="Screen Shot 2022-01-04 at 16 01 54" src="https://user-images.githubusercontent.com/17686203/148087513-ca870b97-d5e9-4c20-b1ee-289d047006d6.png">

You choose the order in which to resolve the combats (first red underline). Then you resolve **a combat** (second red underline), following the six steps (green bar). This includes one of the detachments losing and making a Retreat move, physically moving the models. Then you go to the next combat and repeat the six steps. Everything that happened in the first combat has been resolved and can potentially affect the second combat. In particular, where the losers retreated to and how this affects any 'new' Retreat moves as there may no longer be enough space to retreat into. 

I can't see a way to read the current rules text in a different way. For example, Retreat moves would have to be totally rewritten so that you check the 15 cm distance at the end of the phase, instead of the end of the Retreat move.

.War engines in close combat
====
{blank}[TODO: Need a suitable graphic.]

> In the Assault phase, your Great Gargant is in an assault with the Space Marines.
> Your Great Gargant is undamaged, so its current Damage Capacity value is 18.
> This means that in the subsequent close combat, you roll nine dice to hit the Space Marines.
> Your opponent may still only roll a number of dice equal to the number of Space Marine units that are in base contact -- four dice, in this case.
> ====
> // IJW editing note: I've removed the reference to the Gargant making a charge move, because the original diagram showed a situation that required war engines being able to barge enemy units out of the way - and that's currently just an experimental rule!

Reviewer: Makes sense — although of course we could draw our graphic in a different way so that this isn't an issue. But it's still worth implementing the thrust of your idea here I think — just with some further tweaks for precision and consistency throughout the example.

```suggestion
Your Great Gargant is in close combat with some Space Marine infantry.
It is undamaged, so its current Damage Capacity value is `18`.
This means that you roll nine dice to hit the Space Marine units.

Your opponent may still only roll a number of dice equal to the number of Space Marine units that are in base contact -- four dice, in this case.
====
```
